src/SignalR/clients/ts/signalr/src/Utils.ts

```diff
@@ -181,3 +187,68 @@ export class ConsoleLogger implements ILogger {
         }
     }
 }
+
+/** @private */
+export function getUserAgentHeader(): [string, string] {
+    let userAgentHeaderName = "X-SignalR-UserAgent";
+    if (Platform.isNode) {
+        userAgentHeaderName = "User-Agent";
```

Comment:
Why not just use "X-SignalR-UserAgent" on all platforms?

Reply:
Because "X-SignalR-UserAgent" is a workaround for not being able to set the user-agent in browsers

Reply:
Yeah, I'd rather use the real `User-Agent` header where we can.

Reply:
When do you get an error for setting the User-Agent?

Reply:
Is there a way to try setting the User-Agent and if it fails, fallback to X-SignalR-UserAgent?

Reply:
Looks like in more recent browser versions we should be able to set the User-Agent, it is no longer on the forbidden headers list: https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name

Reply:
I'm back to keeping X-SignalR-User-Agent, this way the browser can provide the OS and browser etc. in the User-Agent header and we can provide our own info in our header.